nbbuild/travis/scripting.sh

```diff
@@ -20,9 +20,9 @@
 set -e
 
 if [ -z "$GRAALVM" ]; then
-  VERSION=19.0.0
+  VERSION=19.3.0
   BASE=graalvm-ce-$VERSION
-  URL=https://github.com/oracle/graal/releases/download/vm-$VERSION/graalvm-ce-linux-amd64-$VERSION.tar.gz
+  URL=https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-$VERSION/graalvm-ce-java8-linux-amd64-vm-$VERSION.tar.gz
```

Comment:
Good. Please enable `scripting.sh` in `.travis.yml` so it really runs.

Reply:
Thanks for the input Jaroslav, I will create one PR and enable `scripting.sh` in `.travis.yml`